unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarName.cs

```diff
@@ -32,6 +33,10 @@ public void SetName(string name)
         }
     }
 
+    public void SetTalking(bool talking) {
```

Comment:
Just in case we should have a 
```
    void OnDisable()
    {
        SetTalking(false);
    }
```

Since the avatars are pooled now, we may have the case of an avatar returning to the pool with the talkingIconContainer turned ON, and when recycling from the pool, the avatar may have it turned ON, so let's turn it off every time the avatar name is disabled (this happens for example when the avatar is disabled to return to the pool)

Reply:
I don't think we need that, since SetTalking is not triggered once. As the name and other properties, they are updated constantly since these properties are synced with the Avatar Shape representation in kernel. See here: https://github.com/decentraland/explorer/pull/1277/files/50ee70993ad10a563f266e9f3f5debb4d0fbcab5#diff-3f1d47987ad32229bc1c30015b196428R105

But I'll add it nonetheless. There may be a small window of inconsistency.